internal/core/data/event_test.go

```diff
@@ -207,8 +211,20 @@ func newAddEventMockDB(persist bool) *dbMock.DBClient {
 
 func TestAddEventWithPersistence(t *testing.T) {
 	reset()
+
+	// no need to mock this since it's all in process
+	msgClient, _ = messaging.NewMessageClient(msgTypes.MessageBusConfig{
+		PublishHost: msgTypes.HostInfo{
+			Host:     "*",
+			Protocol: "tcp",
+			Port:     5563,
+		},
+		Type: "zero",
+	})
```

Comment:
I needed to populate this global variable as the tests would panic without it. Sadly, the test TestMain I removed in device_test.go was populating this and causing this test to be dependent on the execution of that test. This also goes for the chEvents global variable being populated for this test.